packages/opencensus-exporter-stackdriver/src/stackdriver-stats-utils.ts

```diff
@@ -0,0 +1,288 @@
+   * Creates Metric display name.
+   * @param name The metric name
+   * @param displayNamePrefix The metric display name.
+   */
+  static createDisplayName(name: string, displayNamePrefix: string): string {
+    return path.join(displayNamePrefix, name);
+  }
+
+  /**
+   * Converts a OpenCensus Type to a StackDriver MetricKind.
+   * @param metricDescriptorType The kind of metric.
+   */
+  static createMetricKind(metricDescriptorType: MetricDescriptorType):
+      MetricKind {
+    if (metricDescriptorType === MetricDescriptorType.GAUGE_INT64 ||
+        metricDescriptorType === MetricDescriptorType.GAUGE_DOUBLE) {
+      return MetricKind.GAUGE;
+    } else if (
+        metricDescriptorType === MetricDescriptorType.CUMULATIVE_INT64 ||
+        metricDescriptorType === MetricDescriptorType.CUMULATIVE_DOUBLE ||
+        metricDescriptorType === MetricDescriptorType.CUMULATIVE_DISTRIBUTION) {
+      return MetricKind.CUMULATIVE;
+    }
+    return MetricKind.UNSPECIFIED;
+  }
+
+  /**
+   * Converts a OpenCensus Type to a StackDriver ValueType.
+   * @param metricDescriptorType The kind of metric.
+   */
+  static createValueType(metricDescriptorType: MetricDescriptorType):
+      ValueType {
+    if (metricDescriptorType === MetricDescriptorType.CUMULATIVE_DOUBLE ||
+        metricDescriptorType === MetricDescriptorType.GAUGE_DOUBLE) {
+      return ValueType.DOUBLE;
+    } else if (
+        metricDescriptorType === MetricDescriptorType.GAUGE_INT64 ||
+        metricDescriptorType === MetricDescriptorType.CUMULATIVE_INT64) {
+      return ValueType.INT64;
+    } else if (
+        metricDescriptorType === MetricDescriptorType.GAUGE_DISTRIBUTION ||
+        metricDescriptorType === MetricDescriptorType.CUMULATIVE_DISTRIBUTION) {
+      return ValueType.DISTRIBUTION;
+    } else {
+      return ValueType.VALUE_TYPE_UNSPECIFIED;
+    }
+  }
+
+  /**
+   * Constructs a LabelDescriptor from a LabelKey.
+   * @param tag The Tags to get TimeSeries information from.
+   */
+  static createLabelDescriptor(labelKeys: LabelKey[]): LabelDescriptor[] {
+    const labelDescriptorList = labelKeys.map(labelKey => {
+      return {
+        key: labelKey.key,
+        valueType: 'STRING',  // Now we only support String type.
+        description: labelKey.description
+      } as LabelDescriptor;
+    });
+
+    // add default "opencensus_task" label.
+    labelDescriptorList.push({
+      key: StackdriverStatsExporterUtils.OPENCENSUS_TASK,
+      valueType: 'STRING',
+      description: StackdriverStatsExporterUtils.OPENCENSUS_TASK_DESCRIPTION
+    });
+    return labelDescriptorList;
+  }
+
+  /**
+   * Creates a Metric using the LabelKeys and LabelValues.
+   * @param metricDescriptor The OpenCensus MetricDescriptor.
+   * @param labelValues The OpenCensus LabelValue.
+   */
+  static createMetric(
+      metricDescriptor: OCMetricDescriptor, labelValues: LabelValue[],
+      metricPrefix: string): {type: string; labels: {[key: string]: string};} {
```

Comment:
The `{type: string; labels: {[key: string]: string};}` type looks fairly complex, what would you think about making it a first class exported type alias?